### DISCUSSION TITLE
Disable Seq authentication in a test linux machine

### DISCUSSION THREAD

**Discussion by agermalid** [datalust/seq-tickets#1690] [Help] [answered]
Hi,
I accidentally turned on authentication in Seq like [Thomas and Alexander](https://docs.datalust.co/discuss/5f2190c894dc5304e342ee5a) :(
How can I disable authentication again?
it's a test linux machine but I cannot afford to lose all the collected data.
Seq is running in a docker container.

Best regards,
Alexey

**Comment by nblumhardt** [comment, marked as answer]
Hi Alexey,

Restoring from backup - https://docs.datalust.co/docs/backup-and-restore - is the easiest way to go. In Docker you can use init scripts - https://docs.datalust.co/docs/init-scripts - to run the restore command easily.

Let me know if you encounter any difficulties!

Best regards,
Nick
